syft/pkg/cataloger/deb/parse_dpkg_status.go

```diff
@@ -154,7 +150,7 @@ func extractSourceVersion(source string) (string, string) {
 }
 
 // handleNewKeyValue parse a new key-value pair from the given unprocessed line
-func handleNewKeyValue(line string) (string, interface{}, error) {
+func handleNewKeyValue(line string) (key string, val interface{}, err error) {
```

Comment:
`key` is redefined below, this might be confusing?

Reply:
👍🏼 I removed the `var` declaration from the function body